app/src/app/app.module.ts

```diff
@@ -4,13 +4,21 @@ import { BrowserModule } from '@angular/platform-browser';
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
 
+// import ErrorsModule
+// import { ErrorsComponent } from './pages/errors/errors.component';
+import {ShopModule} from "./pages/shop/shop.module";
+import {SharedModule} from "./pages/shop/shared/shared.module";
+
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
   ],
   imports: [
     BrowserModule,
-    AppRoutingModule
+    AppRoutingModule,
+    SharedModule,
+    ShopModule,
```

Comment:
Delete. The module works independently of the app. It will be loaded in the routing.